contracts/tokenization/IStakedToken.sol

```diff
@@ -31,4 +31,6 @@ interface IStakedToken is IERC20 {
      * - `account` must have at least `amount` tokens.
      */
     function burn(address account, uint256 amount) external returns (bool);
+
+    function getRZRPutIn(address delegator, uint256 sAmount) external view returns (uint256);
```

Comment:
Instead of PutIn you can use Deposited.

Reply:
done https://github.com/razor-network/contracts/pull/367/commits/1f97e6752af7f94b48ee399e588abd13ce773033